kafka-connector/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.axonivy.connector.kafka</groupId>
+	<artifactId>kafka-connector</artifactId>
+	<version>10.0.8-SNAPSHOT</version>
+	<packaging>iar</packaging>
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.kafka</groupId>
+			<artifactId>kafka-clients</artifactId>
+			<version>3.4.0</version>
+		</dependency>
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-reload4j</artifactId>
+			<version>1.7.36</version>
+		</dependency>
```

Comment:
could the connector work without reload4j? :thinking:  ... i think our log4j2 stack has a bridge 4 slf4j

Reply:
Strange, at some point it was needed. I started with slf4j-simple, then switched to slf4j-reload4j. Now I removed it and everything seems to work. :-D